### PR TITLE
Fix for some chapters being sent by CR as floats.

### PR DESCRIPTION
**Pull request by NightQuest**
Requires https://github.com/crunchy-labs/crunchyroll-rs/commit/3f3a80f7f7205e8ecb67e15fcf82b988eb88d9b2

### CODE REVIEW

File: crunchy-cli-core/src/utils/download.rs [hunk 1]
```diff
@@ -1225,41 +1225,44 @@ fn write_ffmpeg_chapters(
...
-        writeln!(file, "END={}", event.end)?;
+        writeln!(file, "TIMEBASE=1/1000")?;
+        writeln!(file, "START={}", event.start * 1000u32)?;
+        writeln!(file, "END={}", event.end * 1000u32)?;
         writeln!(file, "title={}", name)?;
 
         last_end_time = event.end;
     }
 
-    // only add a traling chapter if the gab between the end of the last chapter and the total video
```
Comment (bytedream) [inline crunchy-cli-core/src/utils/download.rs:1256]: Haha lol it was something like 4AM when I added this and I maybe were a bit tired

**Review by bytedream — COMMENTED** [review]
I'll approve/merge this when the crunchyroll-rs fixes are released

File: crunchy-cli-core/src/utils/download.rs (hunk 1 above)
Comment (NightQuest) [reply crunchy-cli-core/src/utils/download.rs:1256]: Hah, iktf 🤣

**Review by bytedream — APPROVED** [review]

**Comment by bytedream** [comment]
I merged it locally, GH doesn't seems to recognize this (https://github.com/crunchy-labs/crunchy-cli/commit/f16cd25ea4a40f48b1744c67de09549756ec8505)

**Pull request closed without merging**